here-naksha-lib-core/src/main/java/com/here/naksha/lib/core/models/storage/EWriteOp.java

```diff
@@ -27,8 +27,10 @@
 
 /**
  * The write operations that should be performed.
+ * @deprecated please use {@link naksha.model.request.WriteOp} instead
  */
 @SuppressWarnings("unused")
+@Deprecated
```

Comment:
If it's possible, maybe we should simply delete this?

Reply:
Not to get more repetitive than I am already - this applies to all `@Deprecated` classes

Reply:
I created a backlog task to review these classes later whenever we have personnel resource. For now we will keep them as reference to make sure the new Kotlin classes have everything we need from these.